### ISSUE TITLE
Error when changing mode or using profile

### ISSUE THREAD

**Issue by daved163g** [ledoge/novideo_srgb#62]
Setting the RGB mode initially from the basic tool work, but the screen blanks out briefly.  When when changing back to full gamut by unticking the box the screen flickers (blanks out completely) continuously at a rapid rate. The mouse cursor can be moved but when an attempt is made to grab a window titlebar the move moves itself to the midle of the screen. Initially I found no way to recover from this other than by logging out and back in again using ctl-alt-del. Subsequently I found that invoking Task Manager from the blue screen worked as well, stabilizing the display and allowing the error message to be seen. There was an error on the screen, thus:  NvAPI_GPU_SetColorSpaceConversion failed with error code -1. 
Also, trying to set NoVideo to use a profile in the advanced settings caused the screen to blank completely, other than the mouse. This condition survived both a logout/login, a warm restart and powering the machine off.  I eventually recoveded by booting a different OS (or I could have use safe mode) and disabling the auto start.
Currently using NoVideo on a Dell U2413 off a nVidia Quadro 2000. A second monitor is a Dell 2209W that doesn't need NoVideo. Windows 10, all up to date.

**Comment by daved163g** [comment]
Update: this bug is definitely the result of trying to use a defined (screen!) profile from the Advanced settings. NoVideo puts several error messages on the screen but because these can't be seen and cleared you effectively end up locked out. It is only necessary to use ctl-alt-del then Esc to get the screen back, where the error messages can be cleared. It is then necessary to toggle the mode (off then back on) to get everything stable again. Editing config.xml setting 'use_icc' to false and clearing cc_path to "" also works, then logout and back in.

**Comment by DitherMan420** [comment]
No. A recent nvidia driver broke this. It's 100% due to Nvidia. Now you have to UNCLAMP BEFORE APPLYING HDR, otherwise the clamp stays on even with HDR and it looks messed up.
